test/pipelines/test_eval.py

```diff
@@ -1024,6 +1026,7 @@ def test_reader_eval_in_pipeline(reader):
     assert metrics["Reader"]["f1"] == 1.0
 
 
+@pytest.mark.integration
 @pytest.mark.parametrize("retriever_with_docs", ["tfidf"], indirect=True)
 @pytest.mark.parametrize("document_store_with_docs", ["memory"], indirect=True)
 def test_extractive_qa_eval_document_scope(retriever_with_docs):
```

Comment:
This should be split into several smaller tests - it already has nice blocks which seem good as separate small integration tests

There are several other below with this exact structure, here and in `test_eval_batch.py`: the comment applies to them all